syntax = "proto3";
package server;

message HelloRequest {
    string message = 1;
}

message HelloReply {
    string message = 1;
}

